src/libs/ajax/leonardo/models/runtime-models.ts

```diff
@@ -55,7 +55,7 @@ export interface RuntimeError extends LeoError {
 
 export interface ListRuntimeItem {
   id: number;
-  workspaceId?: string;
+  workspaceId: string | null;
```

Comment:
I am not so sure that requiring workspaceId for `ListRuntimeItem`, but not requiring it for GetRuntimeItem works now. 

Based on the error message i am getting:
```Property 'workspaceId' is missing in type '{ status: LeoRuntimeStatus; id: number; runtimeName: string; googleProject: string; cloudContext: CloudContext; serviceAccount: string; asyncRuntimeFields: AsyncRuntimeFields | null; ... 16 more ...; patchInProgress: boolean; }' but required in type 'ListRuntimeItem'
```

It feels to me that the Runtime = ListRuntimeItem | GetRuntimeItem is wrong now

Reply:
Where are you getting that error?